modules/samples/sample-clients/apim-event-client/src/main/java/org/wso2/carbon/sample/apimevent/DataPublisherUtil.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.sample.apimevent;
+
+import java.io.File;
+
+/**
+ * Datapublisher Util Implementataion.
+ */
+public class DataPublisherUtil {
```

Comment:
please add method level comments.

Reply:
Fixed in 1f69fe0038ac7ea895a7fae7e7714add8fc82281